src/System.Management.Automation/engine/Interop/Windows/CertEnumSystemStore.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1305:FieldNamesMustNotUseHungarianNotation", Justification = "Keep native meth arg names.")]
+    internal static unsafe partial class Windows
+    {
+        [LibraryImport("Crypt32.dll", EntryPoint = "CertEnumSystemStore", SetLastError = true, StringMarshalling = StringMarshalling.Utf16)]
```

Comment:
It seems SetLastError and StringMarshalling can be removed.

Reply:
Why is that? This function seems to set last error so `SetLastError = true` would be needed. It also accepts strings in both the function and callback so I've specified the string marshalling behaviour. While I do have a manual marshal on the `pvSystemStoreLocation` I find it nice to be explicit anyway.

Reply:
Any reason not to just remove the `MarshalAs` and let `StringMarshalling` do its thing? I'm happy to add another comment with an applyable suggestion, just making sure there wasn't more intention there.